app/workers/my_anime_list_list_worker.rb

```diff
@@ -0,0 +1,16 @@
+class MyAnimeListListWorker
+  include Sidekiq::Worker
+
+  def perform(user_id)
```

Comment:
You do realize that the private method can't access the parameter of this, right?  You need to pass it down or set it as an instance variable or property accessor